src/modules/categories/components/Categories.js

```diff
@@ -97,7 +97,7 @@ class Categories extends React.Component<PropsType> {
 
     if (children.length === 0) {
       // last level, our category is a simple page
-      const files = this.getLocalResourceCache(category)
+      const files = this.getFileCache(category)
```

Comment:
This should be named ResourceCache. The term FileCache should no longer exist in the app.
Also "local" is quite correct because it is not the global resource cache of the app.

Reply:
the type is called FileCache.

Reply:
Oh oke you are right.. too many file name changes :sweat_smile: